package.json

```diff
@@ -34,7 +34,8 @@
     "chrome": "node ./lighthouse-cli/manual-chrome-launcher.js",
     "fast": "npm run start -- --disable-device-emulation --disable-cpu-throttling --disable-network-throttling",
     "smokehouse": "node lighthouse-cli/test/smokehouse/smokehouse.js",
-    "deploy-viewer": "cd lighthouse-viewer && gulp deploy"
+    "deploy-viewer": "cd lighthouse-viewer && gulp deploy",
+    "gulp-example": "gulp --gulpfile docs/recipes/gulp/gulpfile.js --cwd docs/recipes/gulp"
```

Comment:
IMO, we leave this as a documented example and not add it to core. Instead, you can create a directory for this recipe (`docs/recipes/gulp-example`) and stick a new package.json in there. Having everything in a single directory will also be future proof. I suspect we'll have a bunch more recipes in the future. Having a folder for each would be good.